tools/Performance/DynamoPerformanceTests/PerformanceTestHelper.cs

```diff
@@ -0,0 +1,132 @@
+        /// </summary>
+        /// <returns></returns>
+        public static BenchmarkDebugConfig getDebugConfig()
+        {
+            return new BenchmarkDebugConfig();
+        }
+
+        /// <summary>
+        /// Get the benchmark debug in process Dynamo debug build config for performance test run
+        /// </summary>
+        /// <returns></returns>
+        public static DebugInProcessConfig getDebugInProcessConfig()
+        {
+            return new DebugInProcessConfig();
+        }
+
+        /// <summary>
+        /// Base config class for regular benchmark job
+        /// </summary>
+        public class DynamoBenchmarkConfig : ManualConfig
+        {
+            /// <summary>
+            /// Minimum count of warmup iterations that should be performed
+            /// </summary>
+            protected int DynamoMinWarmupCount = 6;
+
+            /// <summary>
+            /// Maximum count of warmup iterations that should be performed
+            /// </summary>
+            protected int DynamoMaxWarmuoCount = 9;
+
+            /// <summary>
+            /// Benchmark process will be launched only once
+            /// </summary>
+            protected int DynamoLaunchCount = 1;
+
+            /// <summary>
+            /// Minimum count of target iterations that should be performed.
+            /// </summary>
+            protected int DynamoMinIterationCount = 6;
+
+            /// <summary>
+            /// Maximum count of target iterations that should be performed.
+            /// </summary>
+            protected int DynamoMaxIterationCount = 9;
+
+            public DynamoBenchmarkConfig()
+            {
+                Add(DefaultConfig.Instance.GetLoggers().ToArray()); // manual config has no loggers by default
+                Add(DefaultConfig.Instance.GetExporters().ToArray()); // manual config has no exporters by default
+
+                var defaultColumns = DefaultConfig.Instance.GetColumnProviders().ToList();
+                defaultColumns.RemoveAt(3); // Remove DynamoFilePath column
+                Add(defaultColumns.ToArray());
+                Add(new GraphNameColumn()); // Add Graph Name column
+            }
+        }
+
+        /// <summary>
+        /// Config class that when initialized and used to run the benchmarks
+        /// allows for testing of debug versions of DynamoCore targets.
+        /// </summary>
+        public class BenchmarkDebugConfig : DynamoBenchmarkConfig
+        {
+            public BenchmarkDebugConfig() : base()
+            {
+                Add(JitOptimizationsValidator.DontFailOnError);
+            }
+        }
+
+        /// <summary>
+        /// Config class used to pass command line arguments from the 
+        /// benchmark runner to all benchmarks defined in the test framework class.
+        /// </summary>
+        public class BenchmarkReleaseConfig : DynamoBenchmarkConfig
```

Comment:
Is this necessary as it does not appear to actually do anything?

Reply:
Good point! Maybe I should leave it for now.